DB/api/db.py

```diff
@@ -184,6 +243,23 @@ def perform_query(query, params):
 	finally:
 		if con:
 			con.close()
+def format_energy_list(rows):
+	"""
+	Formats result set for energy_list query
+
+	:param results:
+	:return:
+	"""
```

Comment:
Maybe we can make it even more generic by having the "energy:" JSON tag user selectable?